arm-eventhub/2015-08-01/swagger/EventHub.json

```diff
@@ -780,35 +858,43 @@
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The name of the resource group."
+            "minLength": 1,
+            "maxLength": 90,
+            "description": "The name of the Resource Group."
           },
           {
             "name": "namespaceName",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The namespace name."
+            "minLength": 6,
+            "maxLength": 50,
+            "description": "The Namespace Name within the Resource Group"
           },
           {
             "name": "eventHubName",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The Event Hub name."
+            "minLength": 1,
+            "maxLength": 50,
+            "description": "The EventHub name"
           },
           {
             "name": "authorizationRuleName",
             "in": "path",
             "required": true,
             "type": "string",
-            "description": "The authorization rule name."
+            "minLength": 1,
+            "maxLength": 50,
+            "description": "Authorization rule name."
           },
           {
             "name": "parameters",
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/SharedAccessAuthorizationRuleCreateOrUpdateParameters"
+              "$ref": "#/definitions/SharedAccessAuthorizationRuleResource"
```

Comment:
SharedAccessAuthorizationRuleResource can be renamed to SharedAccessAuthorizationRule